app/models/concerns/annexes/validation.rb

```diff
@@ -2,7 +2,16 @@ module Annexes::Validation
   extend ActiveSupport::Concern
 
   included do
-    # validates :title, presence: true, length: { maximum: 255 }
-    # validates :description, presence: true
+    validates :title, presence: true, length: { maximum: 255 }
+
+    validate  :description_or_image_presence
+  end
+
+  def description_or_image_presence
+    if description.blank? && image_models.empty?
+      message = I18n.t('conclusion_draft_review.errors.not_present_description_and_images')
+      errors.add(:description, message)
+      errors.add(:image_models, message)
```

Comment:
Acá usaría el por defecto:
```suggestion
      errors.add :description, :blank
      errors.add :image_models, :blank
```

Reply:
claro se lo agregue como para que entienda que tiene que tener al menos uno de los dos. Se lo dejo con el :blank

Reply:
Ahh si, quedaría más entendible seguro. Pero en esto que es mega puntual vamos en modo "no metamos nuevas entradas en el lío del I18n"... jeje.

Reply:
jajajaja dale, impecable franco